public_html/wp-content/plugins/wordcamp-attendee-survey/includes/cron.php

```diff
@@ -0,0 +1,183 @@
+<?php
+
+namespace WordCamp\AttendeeSurvey\Cron;
+
+defined( 'WPINC' ) || die();
+
+use CampTix_Plugin;
+
+use function WordCamp\AttendeeSurvey\Email\{get_email_id, queue_survey_email};
+use function WordCamp\AttendeeSurvey\Page\{disable_page,publish_survey_page};
+
+/**
+ * Constants.
+ */
+const DAYS_AFTER_TO_SEND = 2;
+
+/**
+ * Actions & hooks
+ */
+add_action( 'init', __NAMESPACE__ . '\schedule_jobs' );
+add_action( 'wc_attendee_survey_email', __NAMESPACE__ . '\queue_attendee_survey' );
+add_action( 'wc_attendee_disable_survey', __NAMESPACE__ . '\disable_attendee_survey' );
+
+/**
+ * Logs a message to the CampTix email log.
+ */
+function log( $message, $post_id, $data = null ) {
+	/* @var CampTix_Plugin $camptix */
+	global $camptix;
+
+	$camptix->log( $message, $post_id, $data );
```

Comment:
I think it'll be confusing to have logs for a survey plugins stored by a ticketing plugin. There's a general purpose logging function in `mu-plugins/1-logger.php`, and that feels more fitting to me. What do you think?

Reply:
Er, I guess since this is the _attendee_ survey rather than the _organizer_ survey, it isn't that much of a stretch. If they're this closely tied together then I'd lean towards naming the plugin `camptix-attendee-survey` to make that more clear.

Not a big deal either way, though 👍🏻

Reply:
Yeah, because it relies heavily on camptix emails, I thought logging it here makes sense. I'll change the plugin name.